docs/design/autoware-architecture/localization/index.md

```diff
@@ -1,5 +1,300 @@
+
+#### Situations that can make the system unstable
+
+- The vehicle is placed in a structure-less environment, such as a rural landscape, highway, or tunnel
+- Environmental changes have occurred since the map was created, such as snow cover or the construction/destruction of buildings.
+- Surrounding objects are occluded
+- The car is surrounded by objects undetectable by LiDAR, e.g., glass windows, reflections, or absorption (dark objects)
+- The environment contains laser beams at the same frequency as the car's LiDAR sensor(s)
+
+#### Functionality
+
+- The system can estimate the vehicle location on the point cloud map with the error of ~10cm.
+- The system is operable at night.
+
+### 3D-LiDAR or Camera + Vector Map
+
+#### Expected situation
+
+- Road with clear white lines and loose curvatures, such as a highway or an ordinary local road.
+
+#### Situations that can make the system unstable
+
+- White lines are scratchy or covered by rain or snow
+- Tight curvature such as intersections
+- Large reflection change of the road surface caused by rain or paint
+
+#### Functionalities
+
+- Correct vehicle positions along the lateral direction.
+- Pose correction along the longitudinal can be inaccurate, but can be resolved by fusing with GNSS.
+
+### GNSS
+
+#### Expected situation
+
+- The vehicle is placed in an open environment with few to no surrounding objects, such as a rural landscape.
+
+#### Situation that can make the system unstable
+
+- GNSS signals are blocked by surrounding objects, e.g., tunnels or buildings.
+
+#### Functionality
+
+- The system can estimate vehicle position in the world coordinate within an error of ~10m.
+- With RTK attached, the accuracy can be improved to ~10cm
+- The system with this configuration can work without environment maps
+
+### Camera (Visual Odometry, Visual SLAM)
+
+#### Expected situation
+
+- The vehicle is placed in a texture-rich environment, such as urban areas
+
+#### Situations that can make the system unstable
+
+- The vehicle is placed in a Texture-less environment
+- The vehicle is surrounded by other objects.
+- The camera observes significant illumination changes, such as those caused by sunshine, headlights from other vehicles or when approaching the exit of a tunnel.
+- The vehicle is placed in a dark environment.
+
+#### Functionality
+
+- The system can estimate odometry by tracking visual features.
+
+### Wheel speed sensor
+
+#### Expected situation
+
+- The vehicle is running on an ordinary road
+
+#### Situations that can make the system unstable
+
+- The vehicle is running on a slippery or bumpy road, which can cause incorrect observations of wheel speed.
+
+#### Functionality
+
+- The system can acquire the vehicle velocity and estimate distance traveled.
+- The system can estimate more accurate poses at a higher frequency by fusing with external sensors.
+
+<!-- NOTE: Is the second item necessary? Because it is general for any other sensors -->
```

Comment:
Contents of the note need to be actioned and then the note removed.

Reply:
I agree with you. Please remove the "The system can estimate ~" sentence.